www-apps/bugzilla/bugzilla-5.0.6.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit webapp eutils # should not depend on a web server
+
+DESCRIPTION="Bugzilla is the Bug-Tracking System from the Mozilla project"
+SRC_URI="https://ftp.mozilla.org/pub/mozilla.org/webtools/${P}.tar.gz"
+HOMEPAGE="https://www.bugzilla.org"
+
+LICENSE="MPL-2.0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="mysql postgres sqlite test" #modperl graphviz are not suitable for USE as per https://devmanual.gentoo.org/general-concepts/use-flags/
+RESTRICT="!test? ( test )" #why restrict tests ?
+REQUIRED_USE=" || ( mysql postgres sqlite )"
+
+DEPEND="
+	dev-lang/perl
+	>=dev-perl/CGI-3.510.0
+	>=dev-perl/DBI-1.614.0
+	>=dev-perl/TimeDate-2.230.0
+	>=dev-perl/DateTime-0.750.0
+	>=dev-perl/DateTime-TimeZone-1.640.0
+	virtual/perl-Digest-SHA
+	>=dev-perl/Email-MIME-1.904.0
+	>=dev-perl/Email-Sender-1.300.11
+	>=dev-perl/JSON-XS-2.10.0
+	>=dev-perl/List-MoreUtils-0.320.0
+	>=dev-perl/Math-Random-ISAAC-1.0.1
+	>=dev-perl/Template-Toolkit-2.240.0
+	>=dev-perl/URI-1.550.0
+	postgres? ( >=dev-perl/DBD-Pg-1.49 )
+	mysql? ( >=dev-perl/DBD-mysql-4.0.5 )
+	sqlite? ( >=dev-perl/DBD-SQLite-1.290.0 )
+	test? ( dev-perl/Pod-Coverage )
+"
+need_httpd_cgi
+
+src_prepare() {
+	# Get a rid of the bzr files
+	rm -rf .bzr* || die
+
+	# Remove bundled perl modules -> not needed: this directory only contains a README file and is needed for checksetup.pl with sqlite
+#	rm -rf lib/* || die
+
+	# Fix a template error to pass 009bugwords.t
```

Comment:
I wonder if all of these changes would be better as patch files, but that's not necessary -- just a thought.

Have you been able to file any of them upstream/find references?

Reply:
Probably its better as a patch, I also read kent\n dislike sed ^^. I did not file a bug upstream because all of the bugs I could find regarding the test suite seems to be forgotten (https://bugzilla.mozilla.org/buglist.cgi?query_format=advanced&component=Testing%20Suite&resolution=---&product=Bugzilla)